byzcoin/tx_pipeline.go

```diff
@@ -150,9 +160,14 @@ func (s *defaultTxProcessor) CollectTx() ([]ClientTransaction, error) {
 	protocolTimeout := time.After(bcConfig.BlockInterval / 2)
 
 	var txs []ClientTransaction
+	commonVersion := Version(0)
+
 collectTxLoop:
 	for {
 		select {
+		case commonVersion = <-root.CommonVersionChan:
+			// At this point, we only know if other conodes are running the same
```

Comment:
I don't understand the comment, here...

Reply:
Just to say that the value gives a version that is the same for a threshold of conodes but it can be the latest version available so it needs to check that to not create a block to upgrade from version x to x (which is not an upgrade per se)

Reply:
OK, now I understand. I'd put your answer in the comment then ;)